docs/python_docs/python/api/gluon/contrib/index.rst

```diff
@@ -146,27 +147,21 @@ API Reference
 
 .. automodule:: mxnet.gluon.contrib
     :members:
-    :imported-members:
```

Comment:
When do you decide to not include `:imported-members:`?

Reply:
I take out imported members when the package level module also incidentally imports stuff from elsewhere that's already documented. For example here I think this also imports mxnet.ndarray which leads to a ton of warnings during the sphinx build.

Reply:
> I take out imported members when the package level module also incidentally imports stuff from elsewhere that's already documented. For example here I think this also imports mxnet.ndarray which leads to a ton of warnings during the sphinx build.

Ok that's good then, because we need to get warnings back down to zero.